java/gazelle/private/java/java.go

```diff
@@ -34,6 +34,7 @@ var stdlibPrefixes = []types.PackageName{
 	types.NewPackageName("javax.naming"),
 	types.NewPackageName("javax.net"),
 	types.NewPackageName("javax.security"),
+	types.NewPackageName("javax.sql"),
```

Comment:
Is there a programmatic way to check this list is complete for a given JDK?

Reply:
Did a manual update, and included the script to _dump_ packages from a JDK (but not to see if anything is missing).